### PR TITLE
NAS-128564 / 24.10 / Add truenas/sssd to SCALE build

### PR DESCRIPTION
**Pull request by anodos325**
This adds our sssd repository to the build.

### CODE REVIEW

File: conf/build.manifest [hunk 1]
```diff
@@ -195,6 +197,9 @@ apt_preferences:
 - Package: "*cuda*"
   Pin: "version 525.89*"
   Pin-Priority: 1000
+- Package: "*curl*"
```
Comment (yocalebo) [inline conf/build.manifest:200]: This seems unrelated?

Comment (anodos325) [reply conf/build.manifest:200]: runtime requirement of newer curl IIRC.
